src/adapters/database/impl/NeDBAdapter.ts

```diff
@@ -163,6 +169,25 @@ const hasToken = async (token: string): Promise<boolean> => {
     return !!savedToken && savedToken.token === token;
 };
 
+const setTimezone = (id: string, timezone: string): Promise<void> => {
+    Log.debug("Setting timezone:", timezone);
```

Comment:
Log.debug("Setting timezone: ", timezone);

Just adding a space for format

Reply:
The log separates all the arguments with spaces!

When you do a `console.log("1" + "2")` you get `12` because it's one argument
But when you do a `console.log("1", "2")` you get `1 2`!

Reply:
OH amazing sorry ignore then !!! Thank you for explaining